runtime/doc/eval.txt

```diff
@@ -1386,6 +1386,23 @@ v:errors	Errors found by assert functions, such as |assert_true()|.
 <		If v:errors is set to anything but a list it is made an empty
 		list by the assert function.
 
+					*v:event*
+v:event		Data related to an |autocommand| event as a Dictionary. The
+		available keys differ between different events and are
+		specificed at the documentation for each |event|. The possible
+		keys are:
+	regname
```

Comment:
The formatting of this is a bit weird, perhaps make it more similar to what, for example, `:help 'buftype'` or `:help v:fcs_reason` has.